com.microsoft.mrtk.windowsspeech/Subsystems/WindowsSpeechRecognitionSubsystem.cs

```diff
@@ -222,7 +222,8 @@ private void DictationRecognizer_DictationResult(string text, ConfidenceLevel co
             /// <param name="cause">An enumerated reason for the session completing.</param>
             private void DictationRecognizer_DictationComplete(DictationCompletionCause cause)
             {
-                SpeechRecognitionSessionEventArgs eventArgs = new SpeechRecognitionSessionEventArgs(cause.ToString());
+                SpeechRecognitionSessionEventArgs eventArgs = new SpeechRecognitionSessionEventArgs(DictationCompletionCauseToSpeechRecognitionEventReason(cause),
```

Comment:
Wow! "DictationCompletionCauseToSpeechRecognitionEventReason" is a mouthful :smile: